adagrams/game.py

```diff
@@ -1,11 +1,144 @@
+import random
+
 def draw_letters():
-    pass
+    letters_list = []
+    pool_of_letters_dict = {
+        "A": 9,
+        "B": 2,
+        "C": 2,
+        "D": 4,
+        "E": 12,
+        "F": 2,
+        "G": 3,
+        "H": 2,
+        "I": 9,
+        "J": 1,
+        "K": 1,
+        "L": 4,
+        "M": 2,
+        "N": 6,
+        "O": 8,
+        "P": 2,
+        "Q": 1,
+        "R": 6,
+        "S": 4,
+        "T": 6,
+        "U": 4,
+        "V": 2,
+        "W": 2,
+        "X": 1,
+        "Y": 2,
+        "Z": 1
+
+    }
+    letter_freq = {}
+    #letters_list.append(random.choice(list(pool_of_letters_dict.keys())))
+    #return letters_list
+  
+    while len(letters_list) < 10:
+        random_letter = (random.choice(list(pool_of_letters_dict.keys())))
+        if random_letter in letter_freq:
+            letter_freq[random_letter] += 1
+            if letter_freq[random_letter] < pool_of_letters_dict[random_letter]:
+                letters_list.append(random_letter)
+        else:
+            letter_freq[random_letter] = 1
+            letters_list.append(random_letter)
+    return letters_list
```

Comment:
I like the way you used a frequency map and it will work for these particular purposes. However, if we look at the letter pool, we'll notice that each letter has a different number of times it appears. This means that each letter has a different probability of being chosen. To get around this we would have to make some sort of list that holds each letter the number of times it appears in our bank (The extend method is one way you could do this). This would make the rest of the code a little easier because each time we pull a letter we can simply remove it from this larger collection!